### PR TITLE
Tforkner/2909 storybook

### LINKED ISSUE #2909: [Dev] Establish Process For Implementing Storybook w/ Examples
As we refactor and recompose the frontend, we should return to using Storybook to maintain our components. This task is only to get Storybook up and running again, Storybook-ifying components will be an on-going process. As components are worked on, stories should be written for them.

### This task is done when…
 - [ ] Storybook is functioning
 - [ ] 1-3 component have been added to Storybook to use as examples for the other devs

### PR DESCRIPTION
**Pull request by thetif**
Storybook is a tool for creating and cataloging components in React. It provides documentation, encourages reuse, and allows components to be developed in isolation from the system as a whole, which can speed up development in a highly nested component. https://storybook.js.org/tutorials/intro-to-storybook

resolves #2909 

### This pull request changes...

- adds and configures Storybook for our applications
- adds add-ons for a11y, jest, and Figma
- adds package.json scripts for building and starting storybook
- add stories for some components to use as an example

### This is how to verify this change...
- [ ] run storybook locally (needs `npm install`)
- [ ] go to [Our Storybook Github Page](https://cmsgov.github.io/eAPD/?path=/story/example-introduction--page)

### This pull request is ready to merge when...

- [x] Develop has updated automated tests (and all tests are passing)
- [x] This code has been reviewed by someone other than the original author
- [ ] QA has manually tested and created bugs for any issues found (beyond minor fixes)
- [ ] The experience passes a basic manual accessibility audit (keyboard nav, screenreader, text scaling) OR an exemption is documented
- [x] The change has been documented
- [x] Associated OpenAPI documentation has been updated
- [x] Changelog is updated as appropriate

### This feature is done when...

- [x] QA has verified the accessibility and functionality related to the change
- [x] Design has approved the experience
- [x] Product has approved the experience

### CODE REVIEW

**Review by cthulhuplus — APPROVED** [review]
👍 The GHA bits look good to me, I like the emoji!

File: web/src/stories/Introduction.story.mdx [hunk 1]
```diff
@@ -0,0 +1,211 @@
...
+
+  .tip-wrapper code {
+    font-size: 12px;
+    display: inline-block;
+  }
+
+  
+`}</style>
+
+# Welcome to Storybook
```
Comment (tbolt) [inline web/src/stories/Introduction.story.mdx:117]: Is the idea for this page to be updated to have it be customized for eAPD?

**Review by tbolt — APPROVED** [review]
Looks good. It will be good to make "stories" to help test and build components in isolation.

**Review by tbolt — APPROVED** [review]

**Review by radavis — COMMENTED** [review]
:+1:

**Review by radavis — APPROVED** [review]

**Review by beparticular — APPROVED** [review]

**Review by jeromeleecms — APPROVED** [review]
Welcome back to the app, Storybook. Per my conversation with @beparticular this is good to go, a more comprehensive approach to aligning and documenting the app from design -> implementation is currently in discussion and should not be scoped to this issue/PR.

**Comment by mgwalker** [comment]
Ooh, exciting! Hi, eAPD team!

**Comment by thetif** [comment]
Hi Greg! I think I accidentally added you. You still show up as the default reviewer. :D

**Pull request merged**